### PR TITLE
fix(api): Add missing dependencies for API and DB

### PR DESCRIPTION
**Pull request by desertviper**
1. Added a new `health-check.controller` to make it easier to test to make sure the API is running correctly.
1. Added `ConfigService` as a provider in...
    1. `api.module`
    1. `db.module`

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2535,7 +2535,8 @@
         "ansi-regex": {
```
Comment (cakekindel) [inline package-lock.json]: you don't need to change this but in the future you shouldn't need to change package.lock.json if you didn't change the dependencies. To restore without changing the dependencies you can run `npm ci` :smile:

File: src/api/api.module.ts [hunk 2]
```diff
@@ -17,10 +22,12 @@ import { DbModule } from '../db/db.module';
         DbModule,
     ],
     controllers: [
+        HealthCheckController,
```
Comment (cakekindel) [inline src/api/api.module.ts]: awesome!

**Comment by cakekindel** [comment]
Gonna merge, recapped the PR in slack :)

File: package-lock.json (hunk 1 above)
Comment (desertviper) [reply package-lock.json]: Thanks for the info on this!

**Pull request merged**